dqvm/src/worker.lisp

```diff
@@ -192,7 +192,7 @@
     (let ((type mpi:+mpi-byte+)
           (count (* 2 8 (length array))))
       ;; TODO check the mpi-status
-      (mpi::%mpi-recv ptr count type source tag comm mpi:+mpi-status-ignore+))))
+      (mpi::%mpi-recv ptr count type source tag comm (cffi:null-pointer)))))
```

Comment:
is there documentation about this?

Reply:
This is a bit tricky: the MPI standard (see https://www.mpi-forum.org/docs/mpi-3.1/mpi31-report.pdf) says:

> Note that MPI_STATUS_IGNORE is not a special type of MPI_Status object; rather, it is a special value for the argument. In C one would expect it to be NULL, not the address of a special MPI_Status.

In open-mpi, `MPI_STATUS_IGNORE` it is defined in https://github.com/open-mpi/ompi/blob/master/ompi/include/mpi.h.in by:

```#define MPI_STATUS_IGNORE ((MPI_Status *) 0)```

In all C compilers that I'm aware of, that would be equivalent to the NULL pointer but the C standard does not specify that NULL should be a pointer to the address 0.